private/data/i18n/en.json

```diff
@@ -273,20 +273,20 @@
         "price": "Price",
         "optionTitle": "Option",
         "barcode": "Barcode",
-        "compareAtPrice": "Price (MSRP)",
+        "compareAtPrice": "MSRP",
         "fulfillmentService": "Fulfillment service",
         "weight": "Weight",
         "length": "Length",
         "width": "Width",
         "height": "Height",
         "taxable": "Taxable",
-        "taxCode": "Tax Codes",
+        "taxCode": "Tax Code",
         "taxDescription": "Tax Description",
-        "inventoryManagement": "Inventory tracking",
+        "inventoryManagement": "Inventory Tracking",
```

Comment:
@rymorgan Case use in strings is a design choice that I think we should be enforcing some rules around.  I consider `Inventory tracking` to be a correct form versus `Inventory Tracking`, just as I prefer to not see a `:` used on labels.  I'm not asking for a change here, but it's something I think could be covered ASAP in a style guide. (which ever way the case may go ;-) )